### PR TITLE
feat: cascader expandIcon

### LINKED ISSUE #24855: Cascader 开放 expandIcon 的修改
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
需要修改 Cascader 展开项的  icon

### What does the proposed API look like?
开放 expandIcon 属性。

https://github.com/ant-design/ant-design/blob/d6d3de8a1bd0d6b1d6bfa72b33f4bba2f97a3993/components/cascader/index.tsx#L595-L598

### PR DESCRIPTION
**Pull request by xrkffgg**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close https://github.com/ant-design/ant-design/issues/24855

### 💡 Background and solution
- 把原来的 Demo 放出来改了改
效果如下：

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Cascader add `expandIcon` to customize the current item expand icon.           |
| 🇨🇳 Chinese | Cascader 新增 `expandIcon` 来自定义次级菜单展开图标。            |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/cascader/demo/suffix.md](https://github.com/ant-design/ant-design/blob/feat-cascader/components/cascader/demo/suffix.md)
[View rendered components/cascader/index.en-US.md](https://github.com/ant-design/ant-design/blob/feat-cascader/components/cascader/index.en-US.md)
[View rendered components/cascader/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/feat-cascader/components/cascader/index.zh-CN.md)

### CODE REVIEW

**Comment by xrkffgg** [comment]
可以的话，我再更新 snap

File: components/cascader/demo/suffix.md [hunk 1]
```diff
@@ -58,21 +57,28 @@ function onChange(value) {
...
-      style={{ marginTop: '1rem' }}
+      expandIcon={<SmileOutlined />}
       options={options}
       onChange={onChange}
       placeholder="Please select"
     />
-  </div>,
+    <br />
+    <br />
+    <Cascader expandIcon="ab" options={options} onChange={onChange} placeholder="Please select" />
```
Comment (afc163) [inline components/cascader/demo/suffix.md:80]: 不用加 demo，放 test case 里就行。

**Comment by afc163** [comment]
feature branch

File: components/cascader/demo/suffix.md (hunk 1 above)
Comment (xrkffgg) [reply components/cascader/demo/suffix.md:81]: 还是保持 `debug: true` 了

**Review by afc163 — APPROVED** [review]

**Pull request merged**